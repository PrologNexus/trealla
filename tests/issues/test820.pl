:- initialization(main).

main :-
	 X = "abc"||(1*(1+2)), write(X), nl, fail.
main :-
	 X = "abc"||-123, write(X), nl, fail.
main :-
	 X = "abc"||_, write(X), nl, fail.
main :-
	 X = "abc"||"def", write(X), nl, fail.
main :-
	 X = "abc"||f(_), write(X), nl, fail.
main :-
	 X = "abc"||[1,2,3], write(X), nl, fail.
main :-
	 X = "abc"||{1,2,3}, write(X), nl, fail.
main.
