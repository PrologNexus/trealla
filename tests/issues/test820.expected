[a,b,c|1*(1+2)]
[a,b,c|-123]
[a,b,c|_5]
[a,b,c,d,e,f]
[a,b,c|f(_5)]
[a,b,c,1,2,3]
[a,b,c|{1,2,3}]
